static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -303,14 +306,16 @@ $story-package-dynamic-card-text: $story-package-card-colour;
             color: $story-package-dynamic-card-text;
         }
 
-        // ThreeQuarterTall
-        &.fc-item--three-quarters-tall-tablet {
+        // Flip the headline and image!
```

Comment:
We're only doing this on these two layouts for now, we can add more in future PRs.